plugin/source/main.cpp

```diff
@@ -162,6 +163,12 @@ static void quickSelectComboCallback(ConfigItemButtonCombo* item, uint32_t newVa
     WUPS_StoreInt(nullptr, "quickSelectCombo", (int32_t) currentQuickSelectCombination);
 }
 
+static void toggleEmulationComboCallback(ConfigItemButtonCombo* item, uint32_t newValue)
+{
+    currentToggleEmulationCombination = newValue;
+    WUPS_StoreInt(nullptr, "toggleEmulationCombo", (int32_t) currentToggleEmulationCombination);
```

Comment:
You also need to load the value from WUPS storage, otherwise the combination won't be saved. See here:
https://github.com/GaryOderNichts/re_nfpii/blob/52848b9273efc9895dafcac2c613ed6f59b3d616/plugin/source/main.cpp#L91-L93